lib/internal/url.js

```diff
@@ -811,7 +813,13 @@ class URL {
     }
 
     const raiseException = parseSymbol !== kParseURLSymbol;
-    const href = bindingUrl.parse(input, base, raiseException);
+    const pathToFileURL =
+      parseSymbol === kCreateURLFromPosixPathSymbol ||
+      parseSymbol === kCreateURLFromWindowsPathSymbol;
+    const interpretAsWindowsPath = parseSymbol === kCreateURLFromWindowsPathSymbol;
```

Comment:
```suggestion
    const interpretAsWindowsPath = parseSymbol === kCreateURLFromWindowsPathSymbol;
    const pathToFileURL =
      parseSymbol === kCreateURLFromPosixPathSymbol ||
      interpretAsWindowsPath;
```

There's no need to check the equality twice, might-as-well save a few nanoseconds